src/rgw/rgw_qos.h

```diff
@@ -0,0 +1,256 @@
+#pragma once
+#include <chrono>
+#include <thread>
+#include <condition_variable>
+#include "rgw_common.h"
+
+
+class RateLimiter {
+  static constexpr int64_t fixed_float_rgw_ratelimit = 1000;
```

Comment:
please specify the unit here

Reply:
There is no unit in here, it is basically how many digits right of the decimal point (3 digits for 1000)
It is increasing our precision when the bytes or ops limit are too low and the tokens added are 0 because of that.
this is why read_ops() for example is the total ops left / 1000 integral value